enterprise/cmd/cody-gateway/internal/actor/source.go

```diff
@@ -36,6 +36,11 @@ func (e ErrNotFromSource) Error() string {
 
 func IsErrNotFromSource(err error) bool { return errors.As(err, &ErrNotFromSource{}) }
 
+func (e ErrNotFromSource) Is(err error) bool {
+	_, ok := err.(ErrNotFromSource)
```

Comment:
nit: to make it ultra-compatible

```suggestion
	_, ok := errors.Cause(err).(ErrNotFromSource)
```